portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,36 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Ryan!</h1>");
+
+    String json = convertToJson();
+
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  private String convertToJson(){
+    ArrayList<String> surfReport = new ArrayList<String>(3);
+    surfReport.add("low");
+    surfReport.add("60");
+    surfReport.add("3ft");
```

Comment:
This method is actually doing two things:
1) creating an array of hardcoded strings
2) returning those strings as json

I recommend converting this method to taking an ArrayList<String> as an argument and returning the JSON string, since that's more extensible.  This hard-coded string can be created either during construction or in the doGet method.   

Furthermore, I think we can find a better way to store this data that better captures the meaning of these values.  Perhaps it can be represented as an ArrayList<Pair<String, String>> since you're data looks like {(tide, low), (waterTem, 60), (surfHeight, 3ft)}.

Reply:
Do you know a good import statement for the Pair class? I tried using javafx.util.Pair but when it compiles I'm told that it doesn't exist.

Reply:
I think you'd need to add the dependency in pom.xml like this:
https://mvnrepository.com/artifact/org.openjfx/javafx/11

Similar to how you added a dependency for gson.

Alternatively, maybe a HashMap makes more sense in this case since in essence you're storing key/value pairs.